README.md

```diff
@@ -111,11 +111,49 @@ riot_installation_path: /var/www/riot
 # Our public domain name for the Riot Web client
 riot_server_name: "{{ synapse_server_name }}"
 # Look https://github.com/vector-im/riot-web/releases to use the latest version
-riot_version: '1.5.6'
+riot_version: '1.5.15'
+riot_jitsi_preferred_domain: jitsi.riot.im
 # Name to display for the server
 riot_display_name: 'My Org Chat'
-riot_default_theme: light # light or dark
+riot_default_theme: light # 'light', 'dark' or your own 'custom-${theme-name}' (see riot_custom_themes below)
 riot_default_country_code: GB
+
+### Riot UI customization
+riot_customatize_ui: false
+
+riot_welcome_page_template_src: var/www/riot/custom-welcome.html.j2   # Leave it empty if you don't want to overwrite the default Riot welcome page
+riot_welcome_logo_url: welcome/images/logo.svg
+riot_welcome_title: 'Welcome to Riot.im'
+riot_welcome_description: 'Decentralised, encrypted chat &amp; collaboration powered by [matrix]'
+
+riot_custom_branding:
+  welcomeBackgroundUrl: themes/riot/img/backgrounds/valley.jpg
+  authHeaderLogoUrl: themes/riot/img/logos/riot-im-logo-black-text.svg
+  authFooterLinks:
+    - text: blog
+      url: https://blog.riot.im/
+    - text: twitter
+      url: https://twitter.com/@RiotChat
+    - text: github
+      url: https://github.com/vector-im/riot-web
+
+riot_custom_themes:
+  - name: "Deep Purple"
```

Comment:
i prefer led zeppelin but i guess is ok.

Reply:
You can make your own theme then ;) 

I tried to make the Pink Floyd theme but the colors didn't reach me